src/screens/FeedScreen/elements/Search.js

```diff
@@ -86,7 +98,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holyTosca,
+    color: COLORS.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant `COLORS.holytosca` has been renamed from `COLORS.holyTosca`. This change seems fine as long as it is consistent across the entire codebase. If this color is used in multiple places, consider checking all instances to ensure they are updated with the new name.


<!-- This is an auto-generated comment by OSS CodeRabbit -->